src/schema/automobile-insurance.json

```diff
@@ -0,0 +1,139 @@
+{
+    "description": "",
+    "experimental": true,
+    "inputs": {
+        "url": {
+            "typeRef": "#/domains/Generic/types/URL",
+            "description": "Website entry point.",
+            "initial": true
+        },
+        "account": {
+            "typeRef": "#/domains/Generic/types/Account",
+            "description": "Account information for filling in contact details.<br/>Receipts and booking references will typically be sent to specified <code>email</code>.<br/>Some websites also require registering user account, in which case <code>password</code> must be provided.",
```

Comment:
I see same description across all 3 domains, so just remove it from here and move it to `generic.json` instead.